n2y/main.py

```diff
@@ -95,11 +97,19 @@ def export_markdown(client, raw_rows, options):
             if pandoc_output:
                 markdown = pandoc.write(pandoc_output, format='gfm+tex_math_dollars') \
                     .replace('\r\n', '\n')  # Deal with Windows line endings
-                # reformat equations to have one backslash instead of 2
+
+                # normalize target path and create it if it doesn't exist
+                target = options.target
+                if not target[(len(target)-1)] == '/':
+                    target += '/'
+                    options.target = target
+                if not os.path.exists(target):
+                    os.makedirs(target)
+                    options.target = target
 
                 # sanitize file name just a bit
                 # maybe use python-slugify in the future?
-                with open(f"{filename}.md", 'w') as f:
+                with open(f"{options.target}{filename}.md", 'w') as f:
```

Comment:
I would use `os.path.join` here instead of combining the filename with the path like this. Not only is it cleaner but it also will work on different operating systems. See https://docs.python.org/3/library/os.path.html#os.path.join

Reply:
+1 for `os.path.join` There should not be any hardcoded path separators.